app/src/main/java/ru/woodymsk/socialapp/domain/navigation/BottomNavigation.kt

```diff
@@ -0,0 +1,8 @@
+package ru.woodymsk.socialapp.domain.navigation
+
+import ru.woodymsk.socialapp.presentation.common.TabTag
+
+interface BottomNavigation {
```

Comment:
Если это интерфейс, то из названия файла это должно быть понятно. Добавь букву I в начало названия.
```suggestion
interface IBottomNavigation {
```

Reply:
Исправил. 
Всегда стоит так называть файл, интерфейс или в каких-то определённых случаях?

Reply:
В целом встречал 2 подхода: у некоторых интерфейсы просто называются, а  имплементация с `Impl` на конце. У нас в проекте интерфейс `IClassName`, а имплементация просто `ClassName`

Reply:
у меня просто интерфейсы репозиториев называются просто, а их имплементация с приставкой Impl

Reply:
решили оставить BottomNavigation